pyproject.toml

```diff
@@ -78,28 +78,55 @@ build-dir = "build"
+"libafcuda.so",
+"libafcuda.so.3.10.0",
+"libafoneapi.so",
+"libafoneapi.so.3.10.0",
+"libforge.so",
+"libforge.so.1.0.8",
+"libmkl_core.so",
+"libmkl_rt.so",
+"libmkl_intel_thread.so",
+"libmkl_intel_lp64.so",
+]
 
 [tool.scikit-build.cmake]
 build-type = "Release" # not working for whatever reason, duplicated below TODO: fix
 
 [tool.scikit-build.cmake.define]
 # edit to enable/disable building of desired backends
 AF_BUILD_CPU               = "ON"
-AF_BUILD_ONEAPI            = "ON"
-AF_BUILD_CUDA              = "ON"
-AF_BUILD_OPENCL            = "ON"
+#AF_BUILD_ONEAPI            = "ON"
+#AF_BUILD_CUDA              = "ON"
+#AF_BUILD_OPENCL            = "ON"
 AF_BUILD_UNIFIED           = "ON"
 AF_BUILD_DOCS              = "OFF"
 
 CMAKE_BUILD_TYPE           = "Release"
 AF_BUILD_EXAMPLES          = "OFF"
-AF_WITH_IMAGEIO            = "ON"
+#AF_WITH_IMAGEIO            = "ON"
 AF_WITH_LOGGING            = "ON"
 AF_WITH_FMT_HEADER_ONLY    = "ON"
 AF_WITH_SPDLOG_HEADER_ONLY = "ON"
-AF_BUILD_FORGE             = "ON"
+#AF_BUILD_FORGE             = "ON"
 AF_TEST_WITH_MTX_FILES     = "OFF"
 AF_INSTALL_STANDALONE      = "ON"
 
+#AF_WITH_STATIC_CUDA_NUMERIC_LIBS = "OFF"
+#AF_COMPUTE_LIBRARY = "Intel-MKL"
+#AF_ADDITIONAL_MKL_LIBRARIES = "/opt/intel/oneapi/mkl/latest/lib/libmkl_sycl_blas.so.5;/opt/intel/oneapi/mkl/latest/lib/libmkl_sycl_blas.so;/opt/intel/oneapi/mkl/latest/lib/libmkl_sycl_lapack.so.5;/opt/intel/oneapi/mkl/latest/lib/libmkl_sycl_lapack.so;/opt/intel/oneapi/mkl/latest/lib/libmkl_sycl_dft.so;/opt/intel/oneapi/mkl/latest/lib/libmkl_sycl_dft.so.5;/opt/intel/oneapi/mkl/latest/lib/libmkl_sycl_sparse.so.5;/opt/intel/oneapi/mkl/latest/lib/libmkl_sycl_sparse.so;/opt/intel/oneapi/mkl/latest/lib/libmkl_sycl_data_fitting.so.5;/opt/intel/oneapi/mkl/latest/lib/libmkl_sycl_data_fitting.so;/opt/intel/oneapi/mkl/latest/lib/libmkl_sycl_rng.so;/opt/intel/oneapi/mkl/latest/lib/libmkl_sycl_rng.so.5;/opt/intel/oneapi/mkl/latest/lib/libmkl_sycl_stats.so;/opt/intel/oneapi/mkl/latest/lib/libmkl_sycl_stats.so.5;/opt/intel/oneapi/mkl/latest/lib/libmkl_sycl_vm.so;/opt/intel/oneapi/mkl/latest/lib/libmkl_sycl_vm.so.5;/opt/intel/oneapi/mkl/latest/lib/libmkl_intel_ilp64.so;/opt/intel/oneapi/mkl/latest/lib/libmkl_intel_ilp64.so.2;/opt/intel/oneapi/mkl/latest/lib/libmkl_tbb_thread.so;/opt/intel/oneapi/mkl/latest/lib/libmkl_tbb_thread.so.2;/opt/intel/oneapi/mkl/latest/lib/libmkl_core.so.2;/opt/intel/oneapi/mkl/latest/lib/libmkl_core.so;/opt/intel/oneapi/compiler/latest/lib/libimf.so;/opt/intel/oneapi/compiler/latest/lib/libsycl.so.8.0.0;/opt/intel/oneapi/compiler/latest/lib/libsycl.so;/opt/intel/oneapi/compiler/latest/lib/libsycl.so.8;/opt/intel/oneapi/compiler/latest/lib/libsvml.so;/opt/intel/oneapi/compiler/latest/lib/libirng.so;/opt/intel/oneapi/compiler/latest/lib/libintlc.so.5;/opt/intel/oneapi/compiler/latest/lib/libintlc.so;/opt/intel/oneapi/tbb/latest/lib/libtbb.so;/opt/intel/oneapi/tbb/latest/lib/libtbb.so.12.15;/opt/intel/oneapi/tbb/latest/lib/libtbb.so.12"
```

Comment:
These changes shouldn't be part of the default cmake config. Let's revert and keep a local copy instead?